conan/recipes/beman/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+
+
+class bemanRecipe(ConanFile):
+    name = "beman"
+    version = "20240520.0"
+    package_type = "library"
+    requires = ["example/[*]", "scn/[*]"]
+
+    # Optional metadata
+    license = "MIT"
+    author = "Beman Project <mail@bemanproject.org>"
+    url = "https://github.com/beman-project/beman"
+    description = "Organization, documentation, etc. for the Beman Project"
+    # topics = ("<Put some tag here>", "<here>", "<and here>")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    # Sources are located in the same place as this recipe, copy them to the recipe
+    exports_sources = "CMakeLists.txt"
```

Comment:
does this end up overwriting the one from the repo -- I'm guessing no bc this is going into build?

Reply:
Right. This is a different CMakeLists.txt. There's a good chance I can eliminate this altogether. I just need to figure out how to write a conanfile.py that ships nothing is all.

Note that we don't want the top-level CMakeLists.txt in the beman repo because it uses FetchContent, which is a competing dependency resolution technology to packaging systems like Conan.

Reply:
I removed the CMake build for the `beman` Conan package. The CMake support at the top level of the `beman` *repo* still exists, though. It is only useful for CMake dependency management via FetchContent APIs (as opposed to using the Conan package manager to fetch dependencies).